starter_code/src/movies.js

```diff
@@ -2,14 +2,93 @@
 
 // Iteration 1: Ordering by year - Order by year, ascending (in growing order)
 
+
+
+function orderByYear(arr) {
+  let newArr = [...arr];
+  newArr.sort(function(a, b){ 
+    if (a.year - b.year !== 0) {
+      return a.year - b.year;
+      }
+    else  {
+      if (a.title > b.title) {return 1}
+      else if (a.title < b.title) {return -1}
+      else {return 0};
+      }
+  });
+  return newArr;
+}
+orderByYear(movies);
+
 // Iteration 2: Steven Spielberg. The best? - How many drama movies did STEVEN SPIELBERG direct
 
+function howManyMovies(arr) {
+  let newArr = arr.filter(function(element){
+    return (element.director === "Steven Spielberg") && (element.genre.indexOf("Drama")) != -1
```

Comment:
Have a look at the includes() method on the Array.

ex: 

`return (element.director === "Steven Spielberg") && (element.genre.includes("Drama")`